docs/reference/environment-variables.md

```diff
@@ -429,12 +429,12 @@ Directus will attempt to automatically type cast environment variables based on
 ([see above](#type-casting-and-nesting)). If you have a specific need for a given type, you can tell Directus what type
 to use for the given value by prefixing the value with `{type}:`. The following types are available:
 
-| Syntax Prefix | Example                                          | Output                                           |
-| ------------- | ------------------------------------------------ | ------------------------------------------------ |
-| `string`      | `string:value`                                   | `"value"`                                        |
-| `number`      | `number:3306`                                    | `3306`                                           |
-| `regex`       | `regex:/\.example\.com$/`                        | `/\.example\.com$/`                              |
-| `array`       | `array:https://example.com,https://example2.com` | `["https://example.com","https://example2.com"]` |
+| Syntax Prefix | Example                                                                          | Output                                                              |
+| ------------- | -------------------------------------------------------------------------------- | ------------------------------------------------------------------- |
+| `string`      | `string:value`                                                                   | `"value"`                                                           |
+| `number`      | `number:3306`                                                                    | `3306`                                                              |
+| `regex`       | `regex:/\.example\.com$/`                                                        | `/\.example\.com$/`                                                 |
+| `array`       | `array:https://example.com,string:https://example2.com,regex:/\.example3\.com$/` | `["https://example.com","https://example2.com",/\.example3\.com$/]` |
```

Comment:
Slightly unsure on whether this example is enough. Feels like the old `array:https://example.com,https://example2.com` that only showcases `array:` is still valuable for users that only wants it to be cast as an array.

Maybe something like this? But I'm unsure how to achieve that with markdown.
![chrome_49k6AeNLpW](https://user-images.githubusercontent.com/42867097/134458763-c8a5994f-c8ef-4f71-824b-8f1a57debc5b.png)

It might also just be clearer & easier to break it out as another small section/table below this.

Reply:
I don't think you can do a rowspan in markdown tables, but a simple `<br>` might do the trick